db/crud_test.go

```diff
@@ -352,3 +353,333 @@ func TestRevisionStoragePruneTombstone(t *testing.T) {
 	assertTrue(t, base.IsKeyNotFoundError(db.Bucket, err), "Revision should be not found")
 
 }
+
+// Checks for unwanted interaction between old revision body backups and revision cache
+func TestOldRevisionStorage(t *testing.T) {
+
+	db, _ := setupTestDBWithCacheOptions(t, CacheOptions{})
```

Comment:
We'll need to defer `Close()` on the returned bucket to avoid test issues. I missed this in #3768